lib/active_record/connection_adapters/sqlserver/schema_statements.rb

```diff
@@ -407,8 +419,13 @@ def lowercase_schema_reflection_sql(node)
 
         # === SQLServer Specific (View Reflection) ====================== #
 
+        def view_exists?(table_name)
+          identifier = SQLServer::Utils.extract_identifiers(table_name)
+          views.include? identifier.object
+        end
```

Comment:
Does the identifier need to be stored in memory?
```views.include? SQLServer::Utils.extract_identifiers(table_name).object```
For a one liner that doesn't stamp out memory. Since it is only used once.